### PR TITLE
Store Profile Image from ODIC Providers upon registration

### PR DESCRIPTION
**Pull request by jmbrown412**
This PR adds functionality that stores a profile image when one is available from the OIDC Provider. Upon successful registration, an image_url field is saved to the identity table if one is returned from authn. image_url is optional

### CODE REVIEW

**Comment by adamdecaf** [comment]
That's some great code!

File: pkg/client/.openapi-generator/FILES [hunk 1]
```diff
@@ -0,0 +1,55 @@
+README.md
```
Comment (InfernoJJ) [inline pkg/client/.openapi-generator/FILES:1]: Find it odd this got added. Wonder whats up.

File: pkg/client/api/openapi.yaml [hunk 2]
```diff
@@ -1330,12 +1337,16 @@ components:
           nullable: true
           pattern: ^\w+$
           type: string
+        imageUrl:
+          maxLength: 255
+          nullable: true
+          pattern: ^\w+$
```
Comment (InfernoJJ) [inline pkg/client/api/openapi.yaml:1343]: Need to fix this one up so its the `format: url` setting.

File: pkg/client/api_authentication.go [hunk 3]
```diff
@@ -74,6 +75,7 @@ func (a *AuthenticationApiService) Authenticated(ctx _context.Context) (LoggedIn
 
 	localVarBody, err := _ioutil.ReadAll(localVarHTTPResponse.Body)
 	localVarHTTPResponse.Body.Close()
+	localVarHTTPResponse.Body = _ioutil.NopCloser(_bytes.NewBuffer(localVarBody))
```
Comment (InfernoJJ) [inline pkg/client/api_authentication.go:78]: Wonder why this changed too.

**Comment by adamdecaf** [comment]
It's probably a difference in the OpenAPI version used. I've seen changes like that with older versions.

**Comment by InfernoJJ** [comment]
We should figure out how to pin that otherwise we'll end up flip flopping
on changes.

**Pull request merged**